src/runtime/hexagon_host.cpp

```diff
@@ -659,6 +648,35 @@ WEAK const halide_device_interface *halide_hexagon_device_interface() {
     return &hexagon_device_interface;
 }
 
+WEAK void* halide_hexagon_get_support_lib_symbol(void* user_context, const char *name) {
```

Comment:
I'd prefer to call this `halide_hexagon_host_symbol` or something along those lines. Maybe `halide_hexagon_host_get_symbol`?

The actual library is libhalide_hexaogn_host.so, we're basically just defining a replaceable function equivalent to dlsym(libhalide_hexagon_host.so, ...).